moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -179,6 +178,21 @@ class Robot
                         std::vector<double> & joint_point) const;
 
   /**
+   * @brief getJointSolution returns joint solution for cartesian pose.
+   * @brief This function supports custom tool frame for solving IK.
+   * @param pose - desired pose
+   * @param custom_tool_frame - custom frame id
+   * @param timeout - ik solver timeout
+   * @param attempts - number of IK solve attem
+   * @param seed - ik seed
+   * @param joint_point - joint solution for pose
+   * @return true if joint solution found
+   */
+  bool getJointSolution(const Eigen::Affine3d &pose, const std::string& custom_tool_frame,
```

Comment:
Are you overloading the original getJointSolution and getPose?

Reply:
yes. and I'm gonna overload the addTrajPoint(...) and the execute(...) methods as well and test it on our waypoints in the URDF.

Reply:
Am I missing the overloads for `addTrajPoint`?

Reply:
@shaun-edwards ! Will there be any significant impact if we add the default argument(for custom_tool_frame) in the original getJointSolution method instead of overloading the whole method?